package.json

```diff
@@ -21,7 +21,7 @@
     "format": "prettier --write 'src/**/*.{js,jsx,ts,tsx,md,json}' --config ./.prettierrc"
   },
   "dependencies": {
-    "@deriv-com/quill-ui": "^1.10.24",
+    "@deriv-com/quill-ui": "^1.13.22",
```

Comment:
@agrim-deriv is the `reset.scss` file issue resolved from Quill?

Reply:
Yes, the reset.scss is fixed @shafin-deriv , now just need to test it out if the previous issues still persist.